configuration.nix

```diff
@@ -42,6 +43,7 @@ services = {
     '';
   };
   nix-bitcoin.onionServices.clightning.public = true;
+  systemd.services.clightning.serviceConfig.TimeoutStartSec = "300s";
```

Comment:
Why? Should we also set this in nix-bitcoin?

Reply:
clboss slows down start up and can cause time outs.

Reply:
Is this a possible bug in clboss? We should also set this in nix-bitcoin.

Reply:
@ZmnSCPxj are long `DiG` DNS Seed retrieval times normal?

Reply:
No, you might have permission problems though if `dig` is not `suid`.  E.g. on Guix `ping` is not suid but in most distros the kernel disallows non-`root` from doing some of the UDP shenanigans that `ping` needs, so if you use Guix in a foreign distro and try to use the `ping`from Guix you have to `sudo` into the Guix `ping`.  If `dig` needs similar UDP shenanigans (which I suspect, but do not know for sure), and `dig` on Nix is not suid, then that might cause problems, including `dig` stalling on slower paths.

Reply:
Though a slow `dig` should not delay *startup*, BTW.  This was fixed in CLBOSS 0.11A so that the `dig` probe uses `dig --version` instead of `dig`, which should not require network access.  What version are you using?  Probably my fault for not releasing 0.12 yet...

Reply:
Okay.  So what release of CLBOSS are you using here?  The `dig` probe at startup was changed to `dig --version` which should not stall (that much...?) in 0.11A.  0.10 still had this bug.  This was reported in https://github.com/ZmnSCPxj/clboss/issues/49